Central and Eastern European Online Library Journals.js

```diff
@@ -0,0 +1,308 @@
+{
+	"translatorID": "19cef926-c5b6-42e2-a91c-6f2722f8b36d",
+	"label": "Central and Eastern European Online Library Journals",
+	"creator": "Timotheus Kim",
+	"target": "^https://www\\.ceeol\\.com/search",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-06-25 15:07:56"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Timotheus Kim
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('article')) return "journalArticle";
+	else if (getSearchResults(doc, true)) return "multiple";
+	else return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('a.ng-binding');//Z.debug(rows)//a.ng-binding
```

Comment:
This doesn't seem to be the right selector. `.description a` should work.

Reply:
with `.description a` it cannot detect multiple downloads, therefore I have left it as it is.

Reply:
`a.ng-binding` will match every single tag on the page that AngularJS has bound data to. It's an internal bit of state and it isn't stable. `.description a` looks fine on my end. What do search results have to do with downloads?

Reply:
`.description a` looks fine in zotero client, but in Scaffold i got the error "detectWeb did not match". Anyway i changed it.

Reply:
> `a.ng-binding` will match every single tag on the page that AngularJS has bound data to. It's an internal bit of state and it isn't stable. `.description a` looks fine on my end. What do search results have to do with downloads?

@AbeJellinek I have checked this again in Scaffold. With `.description a` it don't detect multiple select. On every journal page AngularJS bound the data, so from my point of view it is quite stable. If you know another css selector that would detect multiple select, just let me know.